pkg/domain/entities/engine.go

```diff
@@ -43,11 +43,13 @@ type PodmanConfig struct {
 	EngineMode     EngineMode       // ABI or Tunneling mode
 	Identities     []string         // ssh identities for connecting to server
 	MaxWorks       int              // maximum number of parallel threads
+	PassPhrase     string           // ssh passphrase for identity for connecting to server
 	RegistriesConf string           // allows for specifying a custom registries.conf
+	Remote         bool             // Connection to API Service "remote"
```

Comment:
Can the comment here be made a little more clear? "This is a connection to a remote Podman instance using the REST API" or similar?